docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -63,15 +58,28 @@ To add a new stage in the review workflows:
 1. Click on the **Add new stage** button.
 2. Write the *Stage name*.
 3. Select a *Color*.
-3. Click on the **Save** button.
+4. Select *Roles* that can change the stage, if the entity is currently in that review stage.
+5. Click on the **Save** button.
 
 By default new stages are appended, but they can be reordered anytime using the ![drag & drop](/img/assets/icons/drag.svg) button.
 
+:::tip
+To set up roles for each stage, you can either click "Apply to all stages" to apply the current roles to all other stages of the workflow or use "Duplicate stage" of the stage context menu.
+of the workflow or use "Duplicate stage" of the stage context menu, to ease the process.
```

Comment:
Is this a mistake?

Reply:
Why do you think that?

Reply:
```suggestion

```

L68 seems to be repeating what's said in L67 🤔 ?

Reply:
Ah thank - GH must have made a mistake when using the commit suggestion. Will fix!

Reply:
Fixed and rebased.